frontend/resourceadm/components/ResourceNameAndId/ResourceNameAndId.tsx

```diff
@@ -107,17 +110,17 @@ export const ResourceNameAndId = ({
     <>
       <Paragraph size='small'>{text}</Paragraph>
       <Label className={classes.label} size='small'>
```

Comment:
We can simplify the code by adding the htmlFor attribute and add a id to the text field. This simplifies the code because of do not need to repeat the aria label twice. :) Please see the next comment.
```suggestion
      <Label className={classes.label} size='small htmlFor={your-text-field-id}'>
```